lib/mumuki/laboratory/locales/datetime.es.yml

```diff
@@ -3,23 +3,23 @@ es:
   datetime:
     distance_in_words:
       about_x_hours:
-        one: Alrededor de 1 hora
-        other: Alrededor de %{count} horas
+        one: aproximádamente 1 hora
+        other: aproximádamente %{count} horas
       about_x_months:
-        one: Alrededor de 1 mes
-        other: Alrededor de %{count} meses
+        one: aproximádamente 1 mes
+        other: aproximádamente %{count} meses
       less_than_x_minutes:
-        one: Hace menos de 1 minuto
-        other: Hace menos de %{count} minutos
+        one: menos de 1 minuto
+        other: menos de %{count} minutos
       x_days:
-        one: Hace 1 dia
-        other: Hace %{count} dias
+        one: 1 día
+        other: '%{count} dias'
       x_minutes:
-        one: Hace 1 minuto
-        other: Hace %{count} minutos
+        one: 1 minuto
```

Comment:
The problem with that is that for amounts different than one it will use numbers and that's beyond our control (without overriding a bunch of nasty active_support code).
It will happen for `en` locale as well

Reply:
I understand, but I'm proposing changing the 1 scenario only - otherwise we'd have to provide translations for every number, I understand.

Like this

```ruby
      x_minutes:
        one: un minuto
        other: '%{count} minutos'
```